src/@types/Metadata.ts

```diff
@@ -47,5 +47,6 @@ export interface MetadataMain {
   }[]
   license: string
   name: string
-  type: 'dataset' | 'algorithm'
+  type: 'dataset' | 'algorithm' | 'all'
```

Comment:
issue(blocking): having `type = 'all'` here feels wrong, since it is no actual ocean metadata type. In that context there are only `'dataset'` and `'algorithm'` right now.
It is probably best to move this to another interface / type or to simply use `'all'` per default if no metadata type is defined